dev-manuals/architecture/general-architure.md

```diff
@@ -0,0 +1,34 @@
+# General architecture
+
```

Comment:
This page could also use a Table of Contents (can be manually created or generated with InteliJ)